DesignMinStack.java

```diff
@@ -0,0 +1,59 @@
+/* package codechef; // don't place package name! */
+
+import java.util.*;
+import java.lang.*;
+import java.io.*;
+
+class MinStack {
```

Comment:
Please error guard pop(), top() & getMin() methods to prevent from operating on an empty stack, the program will crash otherwise.